sklearnex/linear_model/tests/test_logreg.py

```diff
@@ -37,11 +37,10 @@ def prepare_input(X, y, dataframe, queue):
     return X_train, X_test, y_train, y_test
 
 
-@pytest.mark.parametrize(
-    "dataframe,queue",
-    get_dataframes_and_queues(device_filter_="cpu"),
-)
+@pytest.mark.parametrize("dataframe,queue", get_dataframes_and_queues())
 def test_sklearnex_multiclass_classification(dataframe, queue):
+    if queue and queue.sycl_device.is_gpu:
+        pytest.skip("Logistic Regression fit for the GPU sycl_queue is buggy.")

```

Comment:
@avolkov-intel is this skip comment is correct?
Or it is just not implemented yet on GPU?

Reply:
I think its not yet supported, only binary

Reply:
Thank you @ethanglaser ! Will update then

Reply:
Ethan is right, please change comment in pytest.skip